### PR TITLE
build(docker/semantic-release): nodejs/npm dependency to lts version & SemanticRelease to v20

### PR DESCRIPTION
**Pull request by AsifNawaz-cnic**

### CODE REVIEW

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -24,27 +24,22 @@ RUN apt-get update \
...
+RUN apt-get -y install postfix python3-setuptools wget jq sudo git php${PHP_VERSION} php${PHP_VERSION}-xdebug pwgen php${PHP_VERSION}-apcu php${PHP_VERSION}-gd php${PHP_VERSION}-xml php${PHP_VERSION}-mbstring zip unzip php${PHP_VERSION}-zip curl php${PHP_VERSION}-curl
+
 # Cleanup
 RUN apt-get -y autoremove && \
     apt-get -y clean
 
 # Add image configuration and scripts
 ADD ${SCRIPTS_DIR_PATH}/post-create.sh /post-create.sh
-RUN chmod 755 /post-create.sh
+RUN chmod +x /post-create.sh
```
Comment (KaiSchwarz-cnic) [inline .devcontainer/Dockerfile:35]: if you do chmod +x locally and add + commit the change, this line is superfluous.

File: .devcontainer/Dockerfile [hunk 2]
```diff
@@ -71,13 +66,9 @@ RUN usermod --shell /usr/bin/zsh ${USERNAME} \
...
+# zsh and commitizen configuration
 COPY ${CONFIGURATION_DIR_PATH}/.zshrc ${USERHOME}/.zshrc
 COPY ${CONFIGURATION_DIR_PATH}/.czrc ${USERHOME}/.czrc
 RUN chown ${USERNAME}.${USERNAME} ${USERHOME}/.zshrc ${USERHOME}/.czrc \
-    && chmod +x ${USERHOME}/.zshrc ${USERHOME}/.czrc
-
-# Add volumes for the app
-ENTRYPOINT exec top -b
\ No newline at end of file
+    && chmod +x ${USERHOME}/.zshrc ${USERHOME}/.czrc
```
Comment (KaiSchwarz-cnic) [inline .devcontainer/Dockerfile:74]: if you if you do `chmod +x` locally and add + commit the change, this line is superfluous.

**Review by KaiSchwarz-cnic — CHANGES_REQUESTED** [review]
please review

File: .devcontainer/Dockerfile (hunk 1 above)
Comment (AsifNawaz-cnic) [reply .devcontainer/Dockerfile]: thanks for the feedback... Done making changes

**Pull request merged**